core/schema/git.go

```diff
@@ -450,10 +462,6 @@ type tagsArgs struct {
 }
 
 func (s *gitSchema) tags(ctx context.Context, parent dagql.Instance[*core.GitRepository], args tagsArgs) (dagql.Array[dagql.String], error) {
-	if !core.DagOpInContext[core.RawDagOp](ctx) {
-		return DagOp(ctx, s.srv, parent, args, s.tags)
-	}
```

Comment:
There's no reason to dagop cache this. The ID here isn't stable (it'll be mixed in with client/session most of the time) - so this actually just wastes buildkit cache space.